rust/src/examples.rs

```diff
@@ -0,0 +1,214 @@
+to the coordinator in the form of `update` messages. The coordinator will
+internally aggregate these (masked) local models.
+
+**Sum2.** In the Sum2 phase, sum participants compute the sum of masks over all
+the local models, and sends it to the coordinator in the form of `sum2` messages.
+
+Equipped with the sum of masks, the coordinator is able to *unmask* the
+aggregated global model, for the next round.
+
+This short description of the protocol skips over many details, but is
+sufficient for the purposes of this guide. For a much more complete
+specification, see the [white paper](https://www.xain.io/assets/XAIN-Whitepaper.pdf).
+
+# Coordinator
+
+The coordinator is configurable via various settings. The project contains
+various ready-made configuration files that can be used, found under the
+`configs` directory of the repository. Typically they look something like
+the following (in TOML format):
+
+```toml
+[api]
+bind_address = "127.0.0.1:8081"
+
+[pet]
+min_sum_count = 1
+min_update_count = 3
+min_sum_time = 5
+min_update_time = 10
+sum = 0.4
+update = 0.9
+expected_participants = 20
+
+[mask]
+group_type = "Prime"
+data_type = "F32"
+bound_type = "B0"
+model_type = "M3"
+
+[model]
+size = 4
+```
+
+The actual files contain more settings than this, but we mention just the
+selection above because they will be the most relevant for this guide.
+
+## Settings
+
+Going from the top, the [`ApiSettings`] include [`bind_address`], which is the
+address the coordinator should listen on for requests from participants. This
+address should be known to all participants.
+
+The [`PetSettings`] specify various parameters of the PET protocol.
+
+- The most
+important are [`sum`] and [`update`], which are the probabilities assigned to
+the selection of sum and update participants, respectively (note that if a
+participant is selected for both roles, the *sum* role takes precedence).
+
+- The settings [`min_sum_count`] and [`min_update_count`] specify, respectively,
+the minimum number of `sum`/`sum2` and `update` messages the coordinator should
+accept. By default, they are set to the theoretical minimum in order for the
+protocol to function correctly.
+
+- To complement, the settings [`min_sum_time`] and
+[`min_update_time`] specify, respectively, the minimum amount of time (in
+seconds) the coordinator should wait for `sum`/`sum2` and `update` messages. To
+allow for more messages to be processed, increase these times.
+
+- [`expected_participants`] should be set to the expected number of participants
+for the session.
+
+The [`MaskSettings`] determines the masking configuration, consisting of the
+group type, data type, bound type and model type. The [`ModelSettings`] specify
+the size of the model used. Both of these settings should be decided in advance
+with participants, and agreed upon by both.
+
+## Running
+
+The coordinator can be run as follows:
+
+```ignore
+$ git clone git://github.com/xaynetwork/xaynet
```

Comment:
for whatever reason the double slashes cause the following part of the url to be rendered in grey like a comment. not sure if we can do sth about it, maybe a different interpreter than `ignore` works here?

Reply:
hopefully @Robert-Steiner 's suggestion sorts this out